de.metas.vertical.pharma/src/main/sql/postgresql/ddl/pharma_permission_control_report.sql

```diff
@@ -50,7 +52,8 @@ SELECT
 	isCustomer, isVendor, unionType,
 	--parameters
 	(SELECT name FROM C_BPartner WHERE C_BPartner_ID = $1) AS param_bp_name,
-	$2 AS param_manufacturer, $3 AS param_wholesale, $4 AS param_agent, $5 AS param_pharmacie, $6 AS param_veterinary
+	$2 AS param_customer,
```

Comment:
when touching code, please look for ways to improve it..according to teo, we can use parameter names instead of `$n` as of postgresql 9.2 or something..